tests/unit/github/test_github_client.py

```diff
@@ -1,285 +1,118 @@
+import os
+
 import pytest
-import responses
 
 from automata.singletons.github_client import GitHubClient
 
-GITHUB_REQUEST_BASE = "https://api.github.com:443/repos/user/repo"
```

Comment:
What's wrong with all these tests?

Reply:
Locally, every test in this file throws a `requests.exceptions.ConnectionError: Connection refused by Responses - the call doesn't match any registered mock` error, which wasn't being thrown before. Reverting to see if there was a local change that broke this on my end.